File: LAB #1/PythonProject/time_logic.py
y =int(input())

x = y

hours = x // 3600
x %= 3600
print(hours)


minutes = x // 60
x %= 60
print(minutes)

seconds = x

print(seconds)

print(y," Seconds is " , hours," hours, ", minutes," minutes, and ", seconds," seconds " )

File: LAB #1/C++ Project/time_logic.cpp
//
// Created by erenp on 27.02.2026.
//
#include <iostream>
#include <string>
using namespace std;
int main() {
    int time;
    cin >> time;
    int firstTime = time;
    int hours = time / 3600;
    time = time % 3600;
    int minutes = time / 60;
    time = time % 60;
    int seconds = time;

    cout<<firstTime<<" Seconds is "<<hours
    << " Hours, " << minutes << " Minutes, and " << seconds<<" Seconds " <<endl;



    return 0;
}